vnext/Microsoft.ReactNative/Views/ShadowNodeBase.h

```diff
@@ -141,6 +141,10 @@ struct REACTWINDOWS_EXPORT ShadowNodeBase : public ShadowNode {
   double m_border[(int)ShadowEdges::CountEdges] = INIT_UNDEFINED_EDGES;
   double m_cornerRadius[(int)ShadowCorners::CountCorners] = INIT_UNDEFINED_CORNERS;
 
+  // Layout conformance
+  std::optional<bool> m_layoutConformance = std::nullopt;
```

Comment:
nit: Consider keeping an enum value here or raw YGErrata. For now there are only two possibilities, but in Fabric we kept it open to add more.

Reply:
In the Fabric impl, we intentionally only allow the prop on View, instead of any ShadowNode.

Reply:
> nit: Consider keeping an enum value here or raw YGErrata. For now there are only two possibilities, but in Fabric we kept it open to add more.

Fair. I'll save a byte per ShadowNode.

> In the Fabric impl, we intentionally only allow the prop on View, instead of any ShadowNode.

Why do we do this in Fabric? Other than views with defined measuring functions, seems like it would be worth being able to apply this to any component type.